app/views/accounts/actions/_disable_totp.html.slim

```diff
@@ -1,3 +0,0 @@
-= button_to disable_totp_url, method: :delete, class: 'usa-button  usa-button--unstyled' do
-  span.hide = t('account.index.authentication_app')
-  = t('forms.buttons.disable')
```

Comment:
this was slightly different than the ERB file, but the ERB file was updated more recently. I think that when rendering, Rails picks the ERB first so this should be a safe delete